doc/user/pimv6.rst

```diff
@@ -113,6 +113,15 @@ Certain signals have special meanings to *pim6d*.
    notifications to the kernel. This command is vrf aware, to configure for a
    vrf, enter the vrf submode.
 
+.. clicmd:: ipv6 ssmpingd X:X::X:X
+
+   Enable ipv6 ssmpingd configuratiaon.A tool for testing multicast
+   connectivity.A client pings server by sending unicast ssmping query.
+   The server replies with both unicast and multicast ssmping replies.
+   User runs ssmping <S>. Client joins (S,G). Client sends unicast
+   to S and server responds with a response. The IPv6 address is a
+   Unicast address
```

Comment:
Should end with a full stop and unicast in small case to maintain consistency.

Reply:
Ack